website/docs/faqs/cloud-vs-core.md

```diff
@@ -0,0 +1,7 @@
+---
+title: What is the difference between dbt Core, the dbt CLI and dbt Cloud?
+---
+
+dbt Core is the software that takes a [dbt project](dbt-projects) (`.sql` and `.yml` files) and a command and then creates tables/views in your warehouse. dbt Core includes a command line interface (CLI) so that users can execute dbt commands using a terminal program. dbt Core is open source and free to use.
```

Comment:
```suggestion
dbt Core is the software that takes a [dbt project](dbt-projects) (`.sql` and `.yml` files) and a command and then creates tables/views in your warehouse. dbt Core includes a command line interface (CLI) so that users can execute dbt commands using a terminal program. dbt Core is [open source](https://github.com/fishtown-analytics/dbt) and free to use.
```